src/helpers.js

```diff
@@ -145,5 +145,9 @@ var deprecate = Marionette.deprecate = function(message, test) {
   }
 };
 
-deprecate._warn = typeof console !== 'undefined' && (console.warn || console.log) || function() {};
+deprecate._console = typeof console !== 'undefined' ? console : {};
+deprecate._console.warn = deprecate._console.warn || deprecate._console.log || function() {};
```

Comment:
What's wrong with just binding to `console`? I don't think it's really necessary to test that `deprecate._warn` calls `console.warn`, only that `deprecate._warn` is called.


```javascript
deprecate._warn = typeof console !== 'undefined' ? _.bind(console.warn || console.log, console) : function() {};
```